app/src/main/java/is/xyz/mpv/MPVActivity.kt

```diff
@@ -56,6 +57,10 @@ class MPVActivity : Activity(), MPVLib.EventObserver, TouchGesturesObserver {
     private lateinit var gestures: TouchGestures
     private lateinit var audioManager: AudioManager
 
+    private var btnSelected = -1
+    private val colorFocussed = Color.argb(128, 128, 128, 176)
```

Comment:
should probably use a tint like `updatePlaylistButtons` does

Reply:
Done.
`val colorFocused = ContextCompat.getColor(applicationContext, R.color.tint_btn_bg_focused)`